pkg/controlplane/reconcilers/endpointsadapter.go

```diff
@@ -139,42 +123,37 @@ func endpointSliceFromEndpoints(endpoints *corev1.Endpoints) *discovery.Endpoint
 			})
 		}
 
-		if allAddressesIPv6(append(subset.Addresses, subset.NotReadyAddresses...)) {
+		if allAddressesIPv6(subset.Addresses) {
 			endpointSlice.AddressType = discovery.AddressTypeIPv6
 		}
 
-		endpointSlice.Endpoints = append(endpointSlice.Endpoints, getEndpointsFromAddresses(subset.Addresses, endpointSlice.AddressType, true)...)
-		endpointSlice.Endpoints = append(endpointSlice.Endpoints, getEndpointsFromAddresses(subset.NotReadyAddresses, endpointSlice.AddressType, false)...)
+		endpointSlice.Endpoints = append(endpointSlice.Endpoints, getEndpointsFromAddresses(subset.Addresses, endpointSlice.AddressType)...)
 	}
 
 	return endpointSlice
 }
 
 // getEndpointsFromAddresses returns a list of Endpoints from addresses that
 // match the provided address type.
-func getEndpointsFromAddresses(addresses []corev1.EndpointAddress, addressType discovery.AddressType, ready bool) []discovery.Endpoint {
+func getEndpointsFromAddresses(addresses []corev1.EndpointAddress, addressType discovery.AddressType) []discovery.Endpoint {
 	endpoints := []discovery.Endpoint{}
 	isIPv6AddressType := addressType == discovery.AddressTypeIPv6
 
 	for _, address := range addresses {
 		if utilnet.IsIPv6String(address.IP) == isIPv6AddressType {
-			endpoints = append(endpoints, endpointFromAddress(address, ready))
+			endpoints = append(endpoints, endpointFromAddress(address))
 		}
 	}
 
 	return endpoints
 }
 
 // endpointFromAddress generates an Endpoint from an EndpointAddress resource.
-func endpointFromAddress(address corev1.EndpointAddress, ready bool) discovery.Endpoint {
+func endpointFromAddress(address corev1.EndpointAddress) discovery.Endpoint {
+	ready := true
 	ep := discovery.Endpoint{
 		Addresses:  []string{address.IP},
 		Conditions: discovery.EndpointConditions{Ready: &ready},
-		TargetRef:  address.TargetRef,
-	}
-
-	if address.NodeName != nil {
-		ep.NodeName = address.NodeName
 	}
```

Comment:
Is this being removed because it's never populated?

Reply:
correct